letsencrypt/tests/cli_test.py

```diff
@@ -66,12 +66,30 @@ def test_exceptions(self):
         attrs = {'view_config_changes.side_effect' : error}
         self.assertRaises(
             errors.Error, self._call, ['--debug'] + cmd_arg, attrs)
-        self._call(cmd_arg, attrs)
 
         attrs['view_config_changes.side_effect'] = [ValueError]
         self.assertRaises(
             ValueError, self._call, ['--debug'] + cmd_arg, attrs)
-        self._call(cmd_arg, attrs)
+
+    @mock.patch("letsencrypt.cli.sys")
+    def test_handle_exception(self, mock_sys):
+        # pylint: disable=protected-access
+        import StringIO
+        from letsencrypt import cli
+        from letsencrypt import errors
```

Comment:
modules that are not under test (here: `StringIO`, `errors`) should be imported at the top of the file